stellar/README.md

```diff
@@ -68,50 +68,48 @@ stellar contract build
 Deploy the gateway contract
 
 ```bash
-node stellar/deploy-contract.js deploy axelar_gateway --version v1.0.0
+node stellar/deploy-contract.js deploy AxelarGateway --version v1.0.0
 ```
 
 Provide `--estimate-cost` to show the gas costs for the initialize transaction instead of executing it.
 
 ### Operators
 
 ```bash
-node stellar/deploy-contract.js deploy axelar_operators --version v1.0.0
+node stellar/deploy-contract.js deploy AxelarOperators --version v1.0.0
 ```
 
 ### Gas Service
 
 ```bash
-node stellar/deploy-contract.js deploy axelar_gas_service --version v1.0.0
+node stellar/deploy-contract.js deploy AxelarGasService --version v1.0.0
 ```
 
 ### Interchain Token Service
 
 Deploy Interchain Token and Token Manager wasm first.
 
 ```bash
-node stellar/deploy-contract.js deploy interchain_token --version v1.0.0
-node stellar/deploy-contract.js deploy token_manager --version v1.0.0
-node stellar/deploy-contract.js deploy interchain_token_service --version v1.0.0
+node stellar/deploy-contract.js deploy InterchainToken --version v1.0.0
+node stellar/deploy-contract.js deploy TokenManager --version v1.0.0
+node stellar/deploy-contract.js deploy InterchainTokenService --version v1.0.0
 ```
 
 ### Example
 
-Note that example contract should use `--wasm-path` option to deploy contract
-
 ```bash
-node stellar/deploy-contract.js deploy example --wasm-path ../axelar-cgp-stellar/target/wasm32-unknown-unknown/release/stellar_example.optimized.wasm
+node stellar/deploy-contract.js deploy AxelarExample --version v1.0.0
```

Comment:
this doesn't work. v1.0.0 was not deployed, need to use wasm-path here from the release tag

Reply:
we can still call it `AxelarExample` since it doesn't need to match the on-chain name from the point of view of the scripts